### PR TITLE
v1.3.0_SSR-1884 update value and selectedIndex doc on choice components

### PR DESCRIPTION
**Pull request by TomokoMiyake**

### CODE REVIEW

File: docs/document/docs/components/desktop/multichoice.md [hunk 1]
```diff
@@ -29,8 +29,18 @@ Here is a list of properties that can be used for modifying the component:
...
+| selectedIndex *1 | Array\<Number\> | []  | List of index of selected item | If `selectedIndex` is valid, item that has the index number will be selected<br>If `value` is set and `selectedIndex` value exists in `items`, item that has the index number will be selected<br>Will result an error if the value of selectedIndex is not an array |
+
+> *1: You can define duplicated value in value and Item.value. In case defining duplicated value, you can handle selected item using `value` and `selectedIndex` property.<br>
+> Example: When defining `items = [{label: 'Orange', value: 'fruit'}, {label: 'Apple', value: 'fruit'}, {label: 'Carrot', value: 'vegetable'}]`<br>
+> If setting `value` as below:<br>
+> - ['fruit', 'vegetable']: The first and third items will be selected.<br>
+> - ['meat', 'other']: No item will be selected.<br>
+> If don't setting `value` and setting `selectedIndex` as below:<br>
```
Comment (dong0) [inline docs/document/docs/components/desktop/multichoice.md:41]: For better format, I think we need to add a new line before "If..."
```suggestion
>
> If don't setting `value` and setting `selectedIndex` as below:<br>
```

File: docs/document/docs/components/desktop/dropdown.md [hunk 2]
```diff
@@ -24,13 +24,23 @@ Here is a list of properties that can be used for modifying the component:
...
 | Item.label | string | null | Text label for each option | If `Item.label` is unspecified, the value of Item.value is displayed on the UI |
-| Item.value | string | null | Value of each option | Will result an error if there is duplicated value in Item.value |
+| Item.value | string | null | Value of each option | Can set duplicated value in Item.value |
+
+> *1: You can define duplicated value in Item.value. In case defining duplicated value, you can handle selected item using `value` and `selectedIndex` property.<br>
+> Example: When defining `items = [{label: 'Orange', value: 'fruit'}, {label: 'Apple', value: 'fruit'}, {label: 'Carrot', value: 'vegetable'}]`<br>
+> If setting `value` as below:<br>
+> - 'fruit': The first mapped value item in Item.value will be selected.<br>
+> - 'other': No item will be selected.<br>
+> If don't setting `value` and setting `selectedIndex` as below:<br>
```
Comment (dong0) [inline docs/document/docs/components/desktop/dropdown.md:41]: For better format, I think we need to add a new line before "If..."
```suggestion
>
> If don't setting `value` and setting `selectedIndex` as below:<br>
```

File: docs/document/docs/components/desktop/multichoice.md (hunk 1 above)
Comment (TomokoMiyake) [reply docs/document/docs/components/desktop/multichoice.md:41]: I agree and I will fix that.

File: docs/document/docs/components/desktop/dropdown.md (hunk 2 above)
Comment (TomokoMiyake) [reply docs/document/docs/components/desktop/dropdown.md:41]: I agree and will fix that.

**Pull request merged**